api/application.proto

```diff
@@ -43,8 +44,8 @@ message Applications {
 }
 
 message GetApplicationRequest {
-  ApplicationIdentifiers application_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
-  google.protobuf.FieldMask field_mask = 2 [(gogoproto.nullable) = false];
+  ApplicationIdentifiers application_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
+  google.protobuf.FieldMask field_mask = 2 [(gogoproto.nullable) = false, (validate.rules).message.required = true];
```

Comment:
Why is this required? We agreed that an empty field mask for getting is allowed to quickly check existence.

Reply:
It's non-nullable, so it should be required.

Reply:
Non-nullable in Go is not the same as required. The Go default empty is fine and avoids allocating all these objects on the heap. Same goes for slices; `nil` slice is empty in Go, which is fine.